mlflow/protos/databricks_managed_catalog_messages.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto2";
+
+package mlflow.managedcatalog;
+
+import "scalapb/scalapb.proto";
+import "databricks.proto";
+
+option java_package = "com.databricks.api.proto.managedcatalog";
+option java_generate_equals_and_hash = true;
+option (scalapb.options) = {
+  flat_package: true,
+};
+
+message TableInfo {
+
+  // Output-only fields (modified to include only what mlflow needs):
+
+  // [Create,Update:IGN BROWSABLE] Full name of Table, in form of <catalog_name>.<schema_name>.<table_name>
+  optional string full_name = 15;
+  // [Create,Update:IGN] Id of the table.
+  optional string table_id = 22;
+}
+
+message GetTable {
+  // Required. Full name of the Table (from URL).
+  optional string full_name_arg = 1;
+
+  // True if delta metadata should be included in the response.
+  optional bool include_delta_metadata = 3;
```

Comment:
Similarly do we need `include_delta_metadata`? Or ok to omit it and allow delta metadata to not-be returned always when we call this endpoint from the client